RecoTracker/LST/plugins/alpaka/LSTModulesDevESProducer.cc

```diff
@@ -25,29 +25,32 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE {
 
   private:
     edm::ESGetToken<SDL::LSTESHostData<SDL::Dev>, TrackerRecoGeometryRecord> lstESHostToken_;
+    std::string ptCutLabel_;
   };
 
-  LSTModulesDevESProducer::LSTModulesDevESProducer(const edm::ParameterSet& iConfig) : ESProducer(iConfig) {
+  LSTModulesDevESProducer::LSTModulesDevESProducer(const edm::ParameterSet& iConfig)
+      : ESProducer(iConfig), ptCutLabel_(iConfig.getParameter<std::string>("ptCutLabel")) {
     setWhatProduced(this, &LSTModulesDevESProducer::produceHost);
     auto cc = setWhatProduced(this, &LSTModulesDevESProducer::produceDevice);
```

Comment:
the case where both can be produced, add the string in the last argument
```suggestion
    auto cc = setWhatProduced(this, &LSTModulesDevESProducer::produceDevice, ptCutLabel_);
```
... it's somewhat more common to add the record label via `ComponentName` string parameter.
Then the LSTProducer will be able to pick 0.8 or 0.6 or whatever via appropriately configured value of `lstESToken_` (or manually derived from the float ptCut parameter there)